src/query/api/v1/handler/prometheus/common.go

```diff
@@ -241,6 +241,9 @@ func ParseTagValuesToQuery(
 
 	nameBytes := []byte(name)
 	return &storage.CompleteTagsQuery{
+		// NB: necessarily spans the entire timerange for the index.
+		Start:            time.Time{},
+		End:              time.Now(),
```

Comment:
Can you add test to catch this please?

Reply:
It's just setting variables for a struct; since it uses this weird gorilla mux, it's not really plausible to generate a request capable of being read by it easily

Reply:
hmm but it wasn't working before? so we should probably have a test to make sure it doesn't break in the future?

Reply:
These weren't being set; won't really be a useful test

Reply:
These weren't being set; won't really be a useful test

Reply:
hm maybe add an e2e test instead of a unit test, something emulating the behaviour that was failing for users but isn't now?

Reply:
like ideally, that test fails without this PR and works with it.

Reply:
Yeah refactored it a bit to make it nicer and added a couple of tests